tests/FakeItEasy.Tests/NullGuardedAssertionTests.cs

```diff
@@ -26,7 +26,7 @@ public static void UnguardedMethodThatThrowsException(string text)
         [Fact]
         public void Assert_should_throw_when_call_is_null()
         {
-            var exception = Record.Exception(() => ((Expression<Action>)null).Should().BeNullGuarded());
+            var exception = Record.Exception(() => ((Expression<Action>)null!).Should().BeNullGuarded());

```

Comment:
I was about to say "Why not use `Should().BeNullGuarded()`, then I saw the name of the test class...